Makefile

```diff
@@ -53,36 +56,45 @@ build_dir:
 
 test: pep8 test_unit test_functional
 
-test_unit:
-	nosetests tests/unit
+run_test: venv_dev package
```

Comment:
Suggestion: Put the test configuration into a separate _Makefile_ in the test folder and include that makefile from here?